pontoon/teams/views.py

```diff
@@ -205,6 +211,43 @@ def request_item(request, locale=None):
             'user_url': request.build_absolute_uri(user.profile_url)
         }
 
+    # Request teams to be enabled for project
+    elif code and request_type == 'teams':
+        code_list = request.POST.getlist('items[]')
+        project = get_object_or_404(Project, slug=code)
+
+        # Validate projects
+        locale_list = (
+            Locale.objects.available()
+            .filter(code__in=code_list)
+        )
+        if not locale_list:
+            return HttpResponseBadRequest('Bad Request: Non-existent teams specified')
+
+        locales = ''.join('- {} ({})\n'.format(l.name, l.code) for l in locale_list)
+
+        mail_subject = u'Team request for {project} ({slug})'.format(
+            project=project.name, slug=project.slug
+        )
+
+        cc = []
+
+        for locale in locale_list:
+            cc += (
```

Comment:
Hey!
I know that this PR will have to be reviewed by PMs, but I hope that you won't mind if I'll add some notes/comments - All of them all optional. If you feel that you haven't learned anything new from them or you just disagree - that's also okay (and feel free to say it out loud :-).

So, currently this loop will make n number of queries. I think that can be changed into:
```python
# distinct() is required, because some of managers can be assigned to multiple locales.
cc = locale_list.values_list('managers_group__user__email', flat=True).distinct()
```